### PR TITLE
Dev refactor

### PR DESCRIPTION
**Pull request by patiem**

### CODE REVIEW

File: client/src/main/java/gui/HostValidator.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    boolean isIpValid = matcher.matches() || matcherLocal.matches();
+    if(!isIpValid) {
+      LOGGER.log(Level.WARNING, "Closing the game as the host IP is invalid");
+    }
+    return isIpValid && isPortValid(retrievedPort);
+  }
+
+  private static boolean isPortValid(int portNumber) {
+    boolean isPortValid = portNumber > 1023 && portNumber < 65537;
```
Comment (smolamarcin) [inline client/src/main/java/gui/HostValidator.java]: magic numbers

File: client/src/main/resources/fxmls/playBoardEmpty.fxml [hunk 2]
```diff
@@ -72,13 +72,13 @@
     </SplitPane>
     <TextField fx:id="winning" editable="false" layoutX="468.0" layoutY="44.0" prefHeight="85.0" prefWidth="206.0">
         <font>
-            <Font name="Garuda" size="35.0" />
+            <Font name="Garuda" size="30.0" />
         </font>
     </TextField>
     <ImageView fitHeight="92.0" fitWidth="154.0" layoutX="566.0" layoutY="435.0" pickOnBounds="true" preserveRatio="true">
-        <Image url="@../images/bb35.png" />
+        <Image url="@../images/wreck.png" />
```
Comment (smolamarcin) [inline client/src/main/resources/fxmls/playBoardEmpty.fxml]: 👍

File: utils/src/main/resources/Polish.properties [hunk 3]
```diff
@@ -0,0 +1,7 @@
+nameLabel= Imie
```
Comment (smolamarcin) [inline utils/src/main/resources/Polish.properties]: Polish signs

File: client/src/main/java/gui/HostValidator.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
```
Comment (rivalez) [inline client/src/main/java/gui/HostValidator.java]: java doc

File: utils/src/main/java/messages/LanguageVersion.java [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+import java.io.InputStream;
+import java.util.Properties;
+
+public class LanguageVersion {
+
+  private static final String SERVER_CONFIG_FILE = "config.properties";
+  private static String LANGUAGE_CONFIG;
+  private Properties properties;
+
+  private String name;
```
Comment (rivalez) [inline utils/src/main/java/messages/LanguageVersion.java]: try use map. what In situation where you will have 10000 communicates?

**Pull request merged**